src/helpers/user_agent.js

```diff
@@ -1,3 +1,11 @@
 const { userAgent } = window.navigator
 
-export const isHotwireNativeApp = /bridge-components: \[.+\]/.test(userAgent)
+export function appSupportsBridgeComponent(component) {
+    const supportedComponents = userAgent.match(/bridge-components: \[(.*?)\]/)
+
+    if (supportedComponents) {
+      return supportedComponents[1].split(" ").includes(component)
+    } else {
+      return false
+    }
```

Comment:
Indentation seems off here? Should be two spaces.

Reply:
Yep good catch. New machine with wrong editor config 👍